File: android-core/src/test/java/com/mparticle/networking/EndpointTest.java
package com.mparticle.networking;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class EndpointTest {

    @Test
    public void parseEnumTest() {
        for (MParticleBaseClientImpl.Endpoint endpoint: MParticleBaseClientImpl.Endpoint.values()) {
            assertEquals(endpoint, MParticleBaseClientImpl.Endpoint.parseInt(endpoint.value));
        }
    }
}

File: android-core/src/test/kotlin/com/mparticle/networking/EndpointTest.kt
package com.mparticle.networking

import org.junit.Assert
import org.junit.Test

class EndpointTest {
    @Test
    fun parseEnumTest() {
        for (endpoint in MParticleBaseClientImpl.Endpoint.values()) {
            Assert.assertEquals(endpoint, MParticleBaseClientImpl.Endpoint.parseInt(endpoint.value))
        }
    }
}
